theme/styles/snippets/markdown.scss

```diff
@@ -55,58 +62,130 @@
     }
   }
 
-  table { 
-    border-collapse: collapse; 
-    display: block;
-    overflow-x: auto;
-    white-space: nowrap;
+  table {
+    width: 100%;
+    border-collapse: collapse;
+    border-top: 1px solid color('black');
+    border-bottom: 1px solid color('black');
 
-    a {
-      @extend %transition-shorter;
-      &:hover {
-        color: lighten(color('black'), 25%);
+    tr {
+      &:not(:last-of-type) {
+        border-bottom: 1px solid color('black');
       }
-    }
 
-    tbody {
-      display: table;
-      width: 100%;
-    }
+      td {
+        vertical-align: middle;
+        padding: 1.25rem 0;
 
-    tr {
-      &:first-of-type {
-        border-top: 1px solid color('black');
+        @include media('md-up') {
+          padding: 1.6875rem 0;
+        }
+
+        &:first-child {
+          padding-right: 2rem;
+          font-family: $serif;
+          font-size: 1.125rem;
+          line-height: 1.35rem;
+          letter-spacing: 0.0275rem;
+
+          @include media('md-up') {
+            font-size: 3.125rem;
+            line-height: 2.8125rem;
+          }
+        }
+
+        &:last-child {
+          font-family: $sans-serif;
+          @extend %sans-xs;
+
+          @include media('md-up') {
+            font-size: 1.875rem;
+            line-height: 1.6875rem;
+          }
+        }
       }
-
-      border-bottom: 1px solid color('black'); 
     }
+  }
+  
+  /* Product Description Styling */
+  &--style-default {
+    padding-bottom: 3.125rem;
+    
+    @include media ('lg-up') {
+      padding-bottom: 12.5rem;
+    }
+  }
 
-    td { 
-      padding: 0.75rem 0;
+  &--style-default, &--style-black-bg {
+    table {
+      width: calc(100vw - #{$site-padding-x-mobile-double});
 
       @include media('md-up') {
-        padding: 1.25rem 0;
-      }   
+        width: calc(100vw - #{$site-padding-x-desktop-double});
```

Comment:
Any particular reason we are using `vw` instead of `%`?

Reply:
The [markdown container div is `col-9`](https://share.getcloudapp.com/L1uNEJnd) but the table should be full-width of the site, minus the site padding. I think if I put the width as 100%, it would be 100% of `col-9`.. Does that make sense??